Buildings/Controls/OBC/ASHRAE/G36_PR1/TerminalUnits/Validation/ReheatController.mo

```diff
@@ -0,0 +1,139 @@
+within Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Validation;
+model ReheatController "Validate model for controlling VAV terminal box with reheat"
```

Comment:
Similar to previous validation model:
```
In general, it would be good to add text boxes to the model diagram that explain which inputs are control inputs and which inputs we are testing for. For example, opeMod and freProSta set to occupated and stage0, respectively, are control inputs and enable testing for selected conditions, such as duct static pressure.
```